layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot/0005-rpi-Add-autoboot-configuration-in-defconfigs.patch

```diff
@@ -0,0 +1,97 @@
+From 706b40d9326a3dcb82d50370324a5921ca3e24ec Mon Sep 17 00:00:00 2001
+From: Zubair Lutfullah Kakakhel <zubair@resin.io>
+Date: Thu, 14 Jun 2018 12:54:14 +0100
+Subject: [PATCH 5/5] rpi: Add autoboot configuration in defconfigs
```

Comment:
Why is autoboot needed for the switch on uboot? Sounds like you are addressing a different issue which should reside in another PR.

Reply:
Autoboot isn't needed for the switch to u-boot. but this shortens the boot time as the default is something like a 2 second delay in u-boot so I had to mess with the defconfigs anyways.